README.md

```diff
@@ -1,6 +1,6 @@
 # linters-config
 
-## How to use this repo?
+## How to use this repo? 🤔
```

Comment:
[OPTIONAL] I think that ❓ emoji would be better.

_*this change is not necessary to approve this project_